Makefile

```diff
@@ -0,0 +1,24 @@
+#
+# Makefile for building ietf documents using
+# Martin Thomson docker image.
+#
+# You can run the image like the following:
+#
+# $ docker run -ti --rm -v $PWD:/code -w /code --entrypoint /bin/sh martinthomson/i-d-template
```

Comment:
I didn't realize people were doing this, but it's not a bad idea.  I can put an ENTRYPOINT instruction in the image for you to save a little bit of typing.

Reply:
Oh, I realize now that I already considered this: the base image uses `CMD ["/bin/bash"]`, which is exactly what I intended to rely on.